packages/core/src/material/PBRBaseMaterial.ts

```diff
@@ -21,6 +21,12 @@ export abstract class PBRBaseMaterial extends BaseMaterial {
   private static _emissiveTextureProp = Shader.getPropertyByName("u_emissiveSampler");
   private static _occlusionTextureProp = Shader.getPropertyByName("u_occlusionSampler");
 
+  private static _clearcoatProp = Shader.getPropertyByName("u_clearcoat");
```

Comment:
is `clearCoat` not `clearcoat`

Reply:
won't fix

Reply:
Won't fix  reason is what? clearcoat will get error by 'Code Speel Checker' and this is not a word.  `Unity` and `babylon`both call this clearCoat not clearcoat.